Sources/SwiftKuery/ConnectionPoolConnection.swift

```diff
@@ -233,6 +269,8 @@ public class ConnectionPoolConnection: Connection {
         }
     }
     
+    // MARK: Transactions
+
```

Comment:
In the transaction section below - please can you fix the typo rolback instead of rollback in the rollback(onCompletion:) parameter `onCompletion`. (NB this is in 2 places!)

Reply:
In all of the onCompletion parameters below we are missing the word 'the' after the words "execution of", e.g. 
"The function to be called when the execution of ^^^ start transaction command has completed."
"The function to be called when the execution of ^^^ commit transaction command has completed." etc